src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java

```diff
@@ -185,7 +186,7 @@ private TokenCreator lookupTokenCreator(String passPhrase) throws JOSEException
 
   private void handleSignEncryptedToken(RoutingContext ctx) {
     try {
-      logger.debug("Encrypted token signing request from " + ctx.request().absoluteURI());
+      logger.debug("Encrypted token signing request from {}", () -> ctx.request().absoluteURI());
```

Comment:
Not consistent. Other logging statements do not use `() ->`

Reply:
Right. Not elegant. Go through the rest or revert this one?

Reply:
I vote changing this one to be consistent with others. The use of `{}` already makes the string construction delayed in log4j2. See https://logging.apache.org/log4j/2.x/manual/api.html

Reply:
ok.